lib/slappy/listener.rb

```diff
@@ -7,7 +7,7 @@ def initialize(pattern, callback)
     end
 
     def call(event)
-      return unless Time.at(event.ts.to_f) > Slappy.start_time
+      return unless Time.at(event.ts.to_f) > Slappy.client.start_time
```

Comment:
Rails/TimeZone: Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.